cpp/src/community/detail/common_methods.cuh

```diff
@@ -203,16 +209,8 @@ graph_contraction(raft::handle_t const& handle,
                         numbering_indices.size(),
                         new_graph_view.local_vertex_partition_range_first());
 
-  relabel<vertex_t, multi_gpu>(
-    handle,
-    std::make_tuple(static_cast<vertex_t const*>((*numbering_map).begin()),
-                    static_cast<vertex_t const*>(numbering_indices.begin())),
-    new_graph_view.local_vertex_partition_range_size(),
-    labels.data(),
-    labels.size(),
-    false);
-
-  return std::make_tuple(std::move(new_graph), std::move(new_edge_weights));
+  return std::make_tuple(
+    std::move(new_graph), std::move(new_edge_weights), std::move(numbering_map));
```

Comment:
So, what this function basically dose is just calling `coarsen_graph`. Do we really need this additional level of indirection?

```
  auto [new_graph, new_edge_weights, numbering_map] =
    coarsen_graph(handle, graph_view, edge_weights, labels.data(), true);
```
Isn't the code above sufficient to replace this graph_contraction function call? Then, better remove this function.